### DISCUSSION TITLE
✅ 🚀 Feature Request: Nested Elements in Accordion Widget

### DISCUSSION THREAD

**Discussion by beethoak** [elementor/elementor#21630] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I want the possibility to add style and elements in Accordion Widget. Exactly as you do Nested Elements por Tabs, please do the same for Accordion Widget.  

### Describe the solution you'd like

Exactly as you do Nested Elements por Tabs, please do the same for Accordion Widget.

**Comment by nicholaszein** [comment]
Hello there @beethoak!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.15! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
